.circleci/config.yml

```diff
@@ -110,7 +110,7 @@ jobs:
   test-ios:
     macos:
       xcode: 15.1.0 # Specify the Xcode version to use
-    resource_class: macos.x86.medium.gen2
```

Comment:
nit: We don't specify `resource_class` for our other repos and let circle ci choose the default one.

Reply:
The Edge functional tests were at times using 100% of the default image CPU causing test failures. We specify the medium resource class to provide more CPU power for these tests.